README.md

```diff
@@ -78,7 +78,28 @@ If you do not specify a process model, the [typical process](blob/main/src/main/
 
 ## Defining your own process
 
-You can also define your own process model completely, but it has to comply with the following requirements:
+You can also define your own process model completely. As of version 0.0.1+, the benchmark application **automatically discovers and registers job workers for static job types defined in your BPMN files**, making configuration much simpler.
+
+### Automatic Job Type Discovery
+
+The application uses the Zeebe BPMN Model API to parse your BPMN files and automatically extract static service task job types. This means:
+
+- **No manual configuration needed**: Job workers are automatically registered for all static job types found in your BPMN
+- **Supports static job types only**: e.g., `"benchmark-task-1"`, `"benchmark-task-2"`
+- **Ignores dynamic expressions**: Dynamic expressions like `"benchmark-task-" + benchmark_starter_id` are ignored since they are already covered by automatic worker registrations
+- **Backward compatible**: Falls back to configuration-based job types if BPMN parsing fails
+
+For each discovered static job type, the system automatically creates workers for all these variants:
+- `jobType` (e.g., "benchmark-task-1")
+- `jobType + "-" + starterId` (e.g., "benchmark-task-1-starter1") 
+- `jobType + "-completed"` (e.g., "benchmark-task-1-completed")
+- `jobType + "-" + starterId + "-completed"` (e.g., "benchmark-task-1-starter1-completed")
+
+This automatic registration means that dynamic expressions in your BPMN like `"benchmark-task-" + benchmark_starter_id` will work without any additional configuration.
```

Comment:
The additional job types should only be registered for the configured job type as in the original code base but not for every job type found in the BPMN model. Please remove this behavior and just register one worker per discovered job type.